packages/langium/test/grammar/grammar-validator.test.ts

```diff
@@ -315,7 +338,9 @@ describe('Check grammar with primitives', () => {
     const grammar = `
     grammar PrimGrammar
     entry Expr:
-        (Word | Bool | Num | LargeInt | DateObj)*;
+        primitives=Primitive*;
```

Comment:
This grammar was changed [because is also contradicts the proposed solution](https://github.com/eclipse-langium/langium/issues/1246#issuecomment-1785494500).